AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -342,6 +342,7 @@ peacocktv.com##.playback-ad-blocker-notice
 psychic.de#%#//scriptlet("prevent-setTimeout", "test.offsetHeight")
 esoligorsk.by###stopAdBlock
 wikihow.com###contribute_footer
+infranken.de###wl-container
```

Comment:
@BlazDT should we block it?

Reply:
With different rules (see my previous reply), yes. It is no paywall.